src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -328,25 +331,29 @@ def fuzz(self,
 class FuchsiaQemuLibFuzzerRunner(new_process.ProcessRunner, LibFuzzerCommon):
   """libFuzzer runner (when Fuchsia is the target platform)."""
 
+  FUCHSIA_BUILD_REL_PATH = os.path.join('build', 'out', 'default')
+
   SSH_RETRIES = 3
   SSH_WAIT = 2
 
   def __init__(self, executable_path, default_args=None):
     fuchsia_pkey_path = environment.get_value('FUCHSIA_PKEY_PATH')
     fuchsia_portnum = environment.get_value('FUCHSIA_PORTNUM')
-    if not fuchsia_pkey_path or not fuchsia_portnum:
+    fuchsia_resources_dir = environment.get_value('FUCHSIA_RESOURCES_DIR')
+    if (not fuchsia_pkey_path or not fuchsia_portnum or
+        not fuchsia_resources_dir):
       raise fuchsia.errors.FuchsiaConfigError(
-          'FUCHSIA_PKEY_PATH and/or FUCHSIA_PORTNUM was not set')
-    # yapf: disable
-    self.ssh_args = [
-        '-vvv',
-        '-i', fuchsia_pkey_path,
-        '-o', 'StrictHostKeyChecking no',
-        '-o', 'UserKnownHostsFile=/dev/null',
-        '-p', str(fuchsia_portnum),
-        'localhost'
-    ]
-    # yapf: enable
+          ('FUCHSIA_PKEY_PATH, FUCHSIA_PORTNUM, or FUCHSIA_RESOURCES_DIR was '
+           'not set'))
+    self.host = Host.from_dir(
+        os.path.join(fuchsia_resources_dir, self.FUCHSIA_BUILD_REL_PATH))
+    self.device = Device(self.host, 'localhost', fuchsia_portnum)
+    # Fuchsia fuzzer names have the format {package_name}/{binary_name}.
+    package, target = environment.get_value('FUZZ_TARGET').split('/')
```

Comment:
can you add a TODO here, and assign it to me to figure out how to handle the '/' in the binary name? as is, this won't work with a few things.